src/ml_tooling/transformers/pandas_transformers.py

```diff
@@ -34,18 +34,38 @@ def transform(self, X):
 # noinspection PyUnusedLocal
 class FillNA(BaseEstimator, TransformerMixin):
     """
-    Fills NA values with given value
+    Fills NA values with given value or strategy. If no value or strategy are
+    supplied missings are imputed with zero. If both a value and a strategy
+    are supplied the strategy will be used.
     """
 
-    def __init__(self, value):
+    def most_freq(X):
```

Comment:
This should be a static method
```suggestion
    @staticmethod
    def most_freq(X):
```